src/beanahead/rx_txns.py

```diff
@@ -657,8 +657,9 @@ def add_txns(self, end: str | pd.Timestamp = END_DFLT):
         ledger_txns = self.rx_txns + new_txns
 
         # ensure all new content checks out before writting anything
-        content_ledger = compose_new_content("rx", ledger_txns)
-        content_defs = compose_new_content("rx_def", new_defs)
+        name_options = utils.set_root_accounts_context(self.path_ledger_main)
+        content_ledger = compose_new_content("rx", ledger_txns, name_options)
```

Comment:
A third argument is now being passed through although the `compose_new_content` method has not been revised to accept a third argument? Could be that this is handled lower down now and the changes here just need reverting? (same with the next line)

Reply:
Possibly. I'll revisit, it's  probably only a force-push away, since it works locally.Thanks for spotting!